tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
```

Comment:
I was using 5.7.

I still can't repro a failure with this test with 5.6, although I can get an unhandled exception when using Dapper's `Query<int>` followed by `Execute`.